docs/manual/getting-started/index.md

```diff
@@ -1,78 +1,83 @@
+If you enter the password incorrectly three times, you will be presented with a reCAPTCHA challenge to prove that you are 
+a human, and not a password-guessing robot. This feature helps protect your LibreTime installation against brute force 
+attacks.
 
 ![](static/Screenshot466-Recaptcha_login.png)
 
-If activated, you will see a link **Reset password** below the password field, which enables users to obtain a password reminder by email. See the chapter *Preferences* for configuration details.
+If activated, you will see a link **Reset password** below the password field, which enables users to obtain a password 
+reminder by email. See the chapter *Preferences* for configuration details.
 
 ![](static/Screenshot467-Reset_password_link.png) 
 
 The **E-mail** address you enter must match the address stored in the database for your **Username**.
 
 ![](static/Screenshot468-Restore_password.png)
 
-Register Airtime
-----------------
-
-After you have logged in as *admin* for the first time, a pop-up window will ask if you wish to send technical support data about your server to Sourcefabric.
-
-![](static/Screenshot389-Yes_help_Airtime.png)
-
-These details can be viewed by clicking on the **Show me what I am sending** link, which expands a scrolling window. The data helps Sourcefabric engineers resolve any problems with your Airtime installation, as well as count the number of installations worldwide.
-
-<img src="static/Screenshot546-Remind_me_never_250.png" width="535" height="275" />
-
-Sourcefabric has a privacy policy regarding data collection, which you can read by clicking the link to <http://www.sourcefabric.org/en/about/policy/> further down. After checking the **Send support feedback** and **privacy policy** boxes, you can submit the data by clicking the **Yes, help Airtime** button.
-
-This window also offers the opportunity to **Promote my station on Sourcefabric.org** (on the page <http://www.sourcefabric.org/en/airtime/whosusing/>) by checking the box. Fill in the form which will appear with some details about your station. The contact details are only requested for verification purposes, and will not be made available to the public. Click the **Browse** button to select a **Station Logo** image from the file manager on your computer.
-
-![](static/Screenshot391-Promote_my_station.png)
 
 The Master Panel
 ----------------
 
-After the pop-up window is closed, you should now see the **Master Panel**, which is present at the top of every page of the Airtime interface. On the left hand side, the Master Panel displays the details of the **Previous** file played out, the current file playing (with an orange progress bar and time elapsed/time remaining), and the details of the **Next** file due to play. It also displays the name and scheduled time of the current show, with a blue progress bar.
+After the pop-up window is closed, you should now see the **Master Panel**, which is present at the top of every page of
```

Comment:
Remove the now-confusing reference to the pop-up? Maybe just "The **Master Panel** can be found at top of every screen of the LibreTime interface.